ddl/backfilling.go

```diff
@@ -56,8 +60,65 @@ const (
 	typeUpdateColumnWorker     backfillWorkerType = 1
 	typeCleanUpIndexWorker     backfillWorkerType = 2
 	typeAddIndexMergeTmpWorker backfillWorkerType = 3
+
+	// InstanceLease is the instance lease.
+	InstanceLease = 60 // s
 )
 
+// enableDistReorg means whether to enable dist reorg. The default is enable.
+// TODO: control the behavior
+var enableDistReorg = atomicutil.NewBool(false)
```

Comment:
So now it is not use a system variable to enable/disable distributed reorg processing, right?

Reply:
It in https://github.com/pingcap/tidb/pull/37161.